platform-hub-web/src/app/kubernetes-tokens/kubernetes-tokens-form.html

```diff
@@ -0,0 +1,83 @@
+<div class="kubernetes-tokens-form">
+  <md-toolbar md-scroll-shrink>
+    <div class="md-toolbar-tools">
+      <h3>
+        <span ng-if="$ctrl.isNew">New Token</span>
+        <span ng-if="!$ctrl.isNew">Edit "{{$ctrl.tokenData.cluster}}" token</span>
+        for {{$ctrl.user.name}} ({{$ctrl.user.email}})
+      </h3>
+    </div>
+  </md-toolbar>
+
+  <loading-indicator loading="$ctrl.loading || $ctrl.saving"></loading-indicator>
+
+  <md-content layout-padding ng-if="!$ctrl.loading">
+
+    <div flex-sm="100" flex-gt-sm="90" flex-gt-md="70" flex-gt-lg="50" class="md-whiteframe-z2">
+      <md-content layout-padding>
+        <form name="$ctrl.kubernetesTokenForm" role="form" ng-submit="$ctrl.createOrUpdate()">
+
+          <md-input-container class="md-block">
+            <label for="cluster">Cluster:</label>
+            <md-select ng-model="$ctrl.tokenData.cluster" 
+              name="cluster"
+              required
+              placeholder="Select the cluster"
+              ng-disabled="!$ctrl.isNew && $ctrl.tokenData.cluster"
```

Comment:
Ah, you can't edit the cluster once a token has been created? (if so, is this also enforced in the backend?)